core/domain/collection_services.py

```diff
@@ -574,6 +576,29 @@ def get_collection_snapshots_metadata(collection_id):
         collection_id, version_nums)
 
 
+def _get_contributor_ids_for_collection(col_id):
```

Comment:
Yes, this is true! I was focusing only on Explorations. Do we need to make a new map reduce job to get the contributors of collections?